### DISCUSSION TITLE
How to change icon of events with specific words?

### DISCUSSION THREAD

**Discussion by ambernagel** [MMRIZE/MMM-CalendarExt3#127] [Q&A]
I was wanting to change the icons of events with titles that include "birthday". I've gotten the color to change with the following code, but I can't get the icon to change - I've tried ev.symbol and ev.icon. 

```
eventTransformer: (ev) => {
  if (ev.title.search('birthday') > -1) ev.color = 'yellow'
  return ev
}
```

**Comment by eouia** [comment]
Use .symbol instead .color

**Reply by ambernagel** [reply]
I’ve tried .symbol and .icon but neither have worked.

**Reply by Stharani-source** [reply]
Have you find another solution for it? i am having the same issue.

**Reply by Stharani-source** [reply]
nevermind, did more research and found the solution.
add this under MMM-CalendarExt3:

eventTransformer: (ev) => {
  if (ev.title.search('birthday') > -1) ev.color = 'blue'
  return ev
}

add this under main calendar module:
config: {
				broadcastPastEvents: true,
				customEvents: [{keyword: 'birthday', symbol: 'birthday-cake'}],
